x/auth/ante/sigverify.go

```diff
@@ -383,6 +405,27 @@ func (svd SigVerificationDecorator) increaseSequence(tx authsigning.Tx, acc sdk.
 	return acc.SetSequence(acc.GetSequence() + 1)
 }
 
+// authenticateAbstractedAccount computes an AA authentication instruction and invokes the auth flow on the AA.
+func (svd SigVerificationDecorator) authenticateAbstractedAccount(ctx sdk.Context, authTx authsigning.Tx, signer []byte, index int) error {
+	// the bundler is the AA itself.
+	selfBundler, err := svd.ak.AddressCodec().BytesToString(signer)
+	if err != nil {
+		return err
+	}
+
+	infoTx := authTx.(interface {
```

Comment:
is this interface not defined elsewhere?

Reply:
it is not, i did not want to pollute the stack with another interface considering serverv2 TX will end up replacing this.